apps/student/src/app/mypage/onepager/page.tsx

```
@@ -0,0 +1,149 @@
+                className="mb-8"
+              />
+              <p className="text-center text-base text-gray-500">
+                제출한 원페이저 파일이 없어요
+              </p>
+            </div>
+          ) : (
+            <>
+              <div className="space-y-8">
+                {currentOnePagers.map((onePager, index) => (
+                  <button
+                    key={`${onePager.onePagerFormId}-${onePager.submitDate}-${index}`}
```

Biome (2.4.15)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)